include/evmc/evmc.h

```diff
@@ -137,7 +137,7 @@ struct evmc_message
     /**
      * The optional value used in new contract address construction.
      *
-     * Ignored unless kind is EVMC_CREATE2.
+     * Ignored unless kind is EVMC_CREATE2. Not required when invoking evmc_execute_fn().
```

Comment:
Minor suggestion to reword, but it's up to you.
```suggestion
     * Needed only for host to calculate created address when kind is EVMC_CREATE2. Ignored in evmc_execute_fn().
```

Reply:
Thanks.